File: Nível 1/1117 - Validação de Nota/main.py
valoresLidos = 0
media = 0

while valoresLidos != 2:
    valor = float(input("A: "))

    if valor < 0.0 or valor > 10.0:
        print("nota invalida")
    else:
        valoresLidos += 1
        media += valor

calc = media / 2
print(f"media = {calc:.2f}")

File: Nível 1/1117 - Validação de Nota/main.lua
valoresLidos = 0
media = 0

while valoresLidos ~= 2 do 
    valor = tonumber(io.read())

    if valor < 0.0 or valor > 10.0 then
        print("nota invalida")
    else
        valoresLidos = valoresLidos + 1
        media = media + valor
    end
end

calc = media / 2
print(string.format("media = %.2f", calc))